Gems/Compression/Code/Platform/Android/PAL_android.cmake

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+
+set(PAL_TRAIT_COMPRESSION_SUPPORTED TRUE)
+set(PAL_TRAIT_COMPRESSION_TEST_SUPPORTED TRUE)
+set(PAL_TRAIT_COMPRESSION_EDITOR_TEST_SUPPORTED TRUE)
```

Comment:
Shouldn't this be false for non-host platforms?

Reply:
Yes this should be false as well